### DISCUSSION TITLE
how to plot in docstrings (using matplotlib)

### DISCUSSION THREAD

**Discussion by zillionare** [mkdocstrings/mkdocstrings#289] [Q&A] [answered]
is it possible?

**Comment by pawamoy** [comment, marked as answer]
I think so, yes! You'd typically use the [mkdocs-macros-plugin](https://github.com/fralau/mkdocs_macros_plugin/) to define a function that returns a plot in SVG, and call that function in a Markdown page/docstring like `{{ plot() }}` :slightly_smiling_face: 

Could you tell us how you usually use the matplotlib Python library to plot things?
